src/test/java/seedu/tr4cker/model/countdown/EventNameTest.java

```diff
@@ -0,0 +1,60 @@
+package seedu.tr4cker.model.countdown;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.tr4cker.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class EventNameTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new EventName(null));
+    }
+
+    @Test
+    public void constructor_invalidName_throwsIllegalArgumentException() {
+        String invalidName = "";
+        assertThrows(IllegalArgumentException.class, () -> new EventName(invalidName));
+    }
+
+    @Test
+    public void isValidName() {
```

Comment:
maybe can rename this method to `testIsValidName()` so that it sounds more like a test method?